analyses/IL_cd_2020/doc_IL_cd_2020.md

```diff
@@ -0,0 +1,21 @@
+# 2020 Illinois Congressional Districts
+
+## Redistricting requirements
+In Illinois, districts must, under Ill. Const. Art. IV, § 3:
+
+1. be contiguous
+2. have equal populations
+3. be geographically compact
+
+### Interpretation of requirements
+We enforce a maximum population deviation of 0.5%.
+
+## Data Sources
+Data for Illinois comes from the ALARM Project's [2020 Redistricting Data Files](https://alarm-redist.github.io/posts/2021-08-10-census-2020/).
+
+## Pre-processing Notes
+No manual pre-processing decisions were necessary.
+
+## Simulation Notes
+We sample 5,000 districting plans for Illinois.
+To balance county and municipality splits, we create pseudocounties for use in the county constraint, which leads to fewer municipality splits than using a county constraint.
```

Comment:
Let's add a bit more detail here.  See some example text in e.g. the WA docs: https://github.com/alarm-redist/fifty-states/blob/main/analyses/WA_cd_2020/doc_WA_cd_2020.md

When you run `pick_county_muni()` it will tell you which counties are split into munis